### ISSUE TITLE
about adding noise

### ISSUE THREAD

**Issue by qingwang24** [xiph/rnnoise#38]
Hi, this is a great project. I have tried your pretrained model, and it performed well in real-world scenarios.
I collected about 50 hours clean speech data and 3 hours noise data myself. I used your code to generate noisy speech data and to extract input features and output labels. But no matter I trained the model from scratch or by loading your pretrained model, the performance of my trained model is worse than that of your model. So is there any trick to construct the training data?
When listening to the constructed noisy speech, I found that noise in some places is too stong to hear nothing. Is that because the speech gain or noise gain was to big in that places? The speech gain was set to between 0.01 and 10. And the noise gain was set to between 0.03 and 10. It seemed that data overflow occured. What do you think about?
Looking forward to your reply. Thanks.

**Comment by jmvalin** [comment]
The noisy data that gets generated depends on the level of the speech and noise in the data you feed it. It's possible your noise is louder or quieter than what I had, which would generate a different range of SNRs than what I had. Ideally it would auto-calibrate everything, but currently it doesn't.

**Comment by xqustc** [comment]
Thank you for your reply. 
Your noise and speech data is about the same level, right? So could you please give me some advices about generating noisy data? Do I have to choose noise and speech data again?

**Comment by xiaojian10** [comment]
Hey, I have some data at the moment, I also want to train my own model, I would like to ask which code you use to generate noisy voice data files and tags.
